Dockerfile

```diff
@@ -39,13 +30,11 @@ RUN apt-get update && \
     #
     # Download packages
     apt-get update && \
-    apt-get install -y --no-install-recommends php7.2-apcu php7.2-bcmath \
-        php7.2-cli php7.2-curl php7.2-fpm php7.2-gd php7.2-intl php7.2-mysql php7.2-xml \
-        php7.2-zip php7.2-mbstring nodejs yarn && \
+    apt-get install -y --no-install-recommends php7.3-apcu php7.3-bcmath php7.3-cli php7.3-curl php7.3-fpm php7.3-gd php7.3-intl php7.3-mysql php7.3-xml php7.3-zip php7.3-mbstring nodejs yarnpkg && \
```

Comment:
Do you still need PHP?

Reply:
I still launch the `pim:install:assets` command
not sure it's needed
I'll test tomorrow